__tests__/errors.test.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const UsageError = require("../errors").UsageError;
+const onlyInstancesOf = require("../errors").onlyInstancesOf;
+
+describe("onlyInstancesOf", () => {
+  test("catches specified error", () => {
```

Comment:
@saintsebastian this test looks great, but we need also one for the opposite scenario: it should re-throws instances of other errors.